draft-ietf-sidrops-aspa-verification.xml

```diff
@@ -162,48 +162,45 @@
+                </t>
                 <t hangText="Provider:">
-                   The term "provider" is synonymously with "transit provider", see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                  The term "provider" is used synonymously with "transit provider" (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>).
                 </t>
-
                 <t hangText="Customer:">
-                   An autonomous system uses a provider for transit, see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                  An autonomous system that uses a provider for transit (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>).
+                </t>
+                <t hangText="Lateral peer:">
+                The term "lateral peer" is synonymous with "non-transit" or "peer-to-peer" relationship. (<xref target="RFC7908"/>, Section 3.2).
                 </t>
-
                 <t hangText="CAS:">
-                  A Customer AS, see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                  Customer AS (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>).
                 </t>
-
                 <t hangText="PAS:">
-                  A Provider AS, see  <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>.
+                  Provider AS (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="1"/>).
                 </t>
-
-                <t hangText="Route is ineligible:">
-                   The term has the same meaning as in <xref target="RFC4271"/>, i.e., "route is ineligible to be installed in Loc-RIB and will be excluded from the next phase of route selection."
-                </t>
-
                 <t hangText="SPAS:">
-		   A Set of Provider ASes, see <xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="3"/>.
+                  Set of Provider ASes (<xref target="I-D.ietf-sidrops-aspa-profile" sectionFormat="comma" section="3"/>).
                 </t>
-
                 <t hangText="VAP:">
-		   A Validated ASPA Payload, see <xref target="ASPA"/>.
+                  Validated ASPA Payload (<xref target="ASPA"/>).
                 </t>
-
                 <t hangText="VAP-SPAS:">
-		   A Validated ASPA Payload of a set of provider ASes, see <xref target="ASPA"/>.
+                  Set of Provider ASes contained in Validated ASPA Payload (<xref target="ASPA"/>).
+                </t>
+                <t hangText="Provider+:">
+                  The term "Provider+" in the definition of the hop-check function is meant to encompass all three possibilities: Provider, non-transparent RS, or mutual-transit neighbor (<xref target="pair-validation"/>).
```

Comment:
This is based on WGLC feedback. During WGLC, Amir Herzberg emphasized that use of Provider+ is essential. Otherwise, it gets confusing to simply use the term Provider here to represent all flavors such as Provider, RS, and mutual-transit neighbor.

Reply:
I think I mentioned it before, the hop-check function should stay away from such terminology.
It would be better to describe the results of the hop-check function as:
- found, not-found, unknown or no-attestation
- in-set, not-in-set, unknown or no-attestation
- or some other form that is not connected to the term provider or AS

Hop-check is a simple lookup function with 3 possible results that checks if `provider_as` is an attested provider of `customer_as` using the present VAP data.

```
hop_check(customer_as, provider_as)
```
a) if there is no entry for `customer-as` in the `VAP` database return `no-attestation`
b) if there is an entry for `provider-as` in the `SPAS` of the `customer_as` `VAP` return `found`
c) else return `not-found`

Why do we need to introduce yet another terminology and abstraction around `Provider+` that needs to be explained? I think in Section 5 about half of the text should be removed.

Reply:
@cjeker 
I will make changes in Section 5 per your suggestions. I will provide much simplified/shorter text in Section 5. However, it is a bit hard to replace 'Provider+' and 'Not Provider+' with something else that is as intuitive. Because we need two 'Not Provider' hops (left and right) in opposite direction to detect Invalid. See the new Sec. 7 (Fig. 2) that I sent in email today. With 'Not Provider+' notation (connotes P2C or P2P) the explanation seems more intuitive for the reader. I will try to think a bit more here.

Reply:
I see not that much of an issue with using different terms:
```
Because we need two 'not-in-set' hops (left and right) in opposite direction to detect Invalid.
```

Will look at the new branch and comment there.